e2e/cephfs.go

```diff
@@ -312,6 +313,36 @@ var _ = Describe("cephfs", func() {
 					}
 				})
 			}
+			By("verify generic ephemeral volume support", func() {
+				// generic ephemeral volume support is beta since v1.21.
+				if !k8sVersionGreaterEquals(f.ClientSet, 1, 21) {
+					Skip("generic ephemeral volume only supported from v1.21+")
+				}
+				err := createCephfsStorageClass(f.ClientSet, f, true, nil)
+				if err != nil {
+					e2elog.Failf("failed to create CephFS storageclass with error %v", err)
```

Comment:
`failed to create CephFS storageclass with error %v` to `failed to create CephFS storageclass: %v` `with error` formatting needs to be adjusted in other places also